demisto_sdk/commands/common/errors.py

```diff
@@ -972,8 +972,8 @@ def breaking_backwards_arg_changed(cls):
     @classmethod
     @error_code_decorator
     def breaking_backwards_command_arg_changed(cls, commands_ls):
-        error_msg = "{}, Your updates to this file contains changes to a name or an argument of an existing command(s).\n" \
-            "Please undo you changes to the following command(s):\n".format(cls.BACKWARDS)
+        error_msg = "{}, Your updates to this file contains changes to a name or an argument of an existing command(" \
+                    "s).\nPlease undo you changes to the following command(s):\n".format(cls.BACKWARDS)
         error_msg += '\n'.join(commands_ls)
```

Comment:
is this on purpose?

Reply:
Yes, it was needed for pre-commit checks. changed it so It looks a bit better :)